platforms-serverless/serverless-framework-lambda/test.ts

```diff
@@ -9,11 +9,13 @@ async function main() {
   console.log({ data: data.$response.data })
 
   const actual = (data.$response.data as any).Payload
-  const binaryString = `,"files":["index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","schema.prisma"]`
+  const binaryString = process.env.PRISMA_FORCE_NAPI === 'true'
+    ? `,"files":["index-browser.js","index.d.ts","index.js","libquery_engine_napi-debian-openssl-1.1.x.so.node","libquery_engine_napi-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
```

Comment:
You can get to this:
```suggestion
    ? `,"files":["index-browser.js","index.d.ts","index.js","libquery_engine_napi-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
```
by adding this to `serverless.yml` in this project at the bottom:
```
    - '!node_modules/.prisma/client/libquery_engine_napi-*'
    - 'node_modules/.prisma/client/libquery_engine_napi-rhel-openssl-1.0.x.so.node'
```